cohort-evaluator-spark/pom.xml

```diff
@@ -199,11 +199,6 @@
 			<artifactId>commons-io</artifactId>
 		</dependency>
 		
-		<dependency>
```

Comment:
Removing this as it's a duplicate in the file, there was a warning during the build